drive/modules/listener/watch_schedular.bal

```diff
@@ -51,10 +51,14 @@ class Job {
     public isolated function execute() {
         error? err = self.registerWatchChannel();
         if (err is error) {
-            log:printWarn(WARN_CHANNEL_REGISTRATION, 'error = err);
+            if (self.retryCount == 1){
+                log:printInfo(CHANNEL_REGISTRATION_IN_PROGRESS);
```

Comment:
We need only very first time. But seems this will be printed every retry as the first log.

Reply:
Yes. That true. We need to refator this